### PR TITLE
Sending 5mm SDL to Minter contract

### PR DESCRIPTION
**Pull request by hannespfeiffer**
SDL on Minter is running low. This tx refills the minter with 5mm SDL

### CODE REVIEW

File: scripts/2022_08_08_send_5mm_SDL_to_minter.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    # Target address
+    minter_address = SDL_MINTER_ADDRESS[CHAIN_IDS[TARGET_NETWORK]]
+
+    # Send sdl to minter
+    sdl.transfer(minter_address, amount_to_send)
+
+    # combine history into multisend txn
+    safe_tx = multisig.multisend_from_receipts()
+    safe_nonce = 54
```
Comment (hammeiam) [inline scripts/2022_08_08_send_5mm_SDL_to_minter.py:34]: any reason to set this as a variable if it's only used once?

**Review by hammeiam — APPROVED** [review]

File: scripts/2022_08_08_send_5mm_SDL_to_minter.py (hunk 1 above)
Comment (penandlim) [reply scripts/2022_08_08_send_5mm_SDL_to_minter.py:34]: +1. Should be ok to set it directly.

**Review by penandlim — APPROVED** [review]
LGTM

**Comment by penandlim** [comment]
Queued up at nonce 54.
https://gnosis-safe.io/app/eth:0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE/transactions/multisig_0x3F8E527aF4e0c6e763e8f368AC679c44C45626aE_0x26ecf6f8405f7d035fa314b8a1491c70d28070019b1f0d1a2c8a4aa5352fc42f

**Pull request merged**